.gitignore

```diff
@@ -21,4 +21,5 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
+# sass to css
```

Comment:
just curious if we want to clear out the defaults in the .gitignore that aren't relevent to our use case

Reply:
seems good